packages/runtime/src/widgets/Form/MultiSelect.tsx

```diff
@@ -264,6 +268,7 @@ const MultiSelect: React.FC<MultiSelectProps> = (props) => {
             notFoundContent="No Results"
             onChange={handleChange}
             onSearch={handleSearch}
```

Comment:
Why do we need both handleSearch and filter options?

Reply:
<img width="1225" alt="image" src="https://github.com/EnsembleUI/ensemble-react/assets/31651612/04d5be8f-58b0-4f31-aa02-23c5cf1f410c">

As per the previous implementation inside the Multiselect widget, when there is no matching option found and user allow to create custom option, we want to show something like this in UI, that's why we need handleSearch

Reply:
Can we leave a comment about this?